ecommerce/frontend/src/components/pages/SignupPage.js

```diff
@@ -1,3 +1,12 @@
+import { SignUpForm } from '../SignUpForm'
+import './SignupPage.less'
 export const SignupPage = () => {
-    return 'signup'
+    return <>
+        <div className="signup-container">  
+            <div className="signup-header">
+                <h1>Signup to Getflix!</h1>
+            </div>  
+            <SignUpForm />
+        </div>
+    </>
```

Comment:
nitpick: no need for the `<>` since there is only one root tag `div`

Reply:
thanks, i had more tags in there before. corrected it, closing.